Source/MediaInfo/Image/File_Tiff.cpp

```diff
@@ -64,22 +64,77 @@ namespace Tiff_Tag
     const int16u ResolutionUnit             = 296;
     const int16u Software                   = 305;
     const int16u ExtraSamples               = 338;
+    const int16u JPEGTables                 = 347;
+    const int16u JPEGProc                   = 512;
+    const int16u JPEGInterchangeFormat      = 513;
+    const int16u JPEGInterchangeFormatLngth = 514;
+    const int16u JPEGRestartInterval        = 515;
+    const int16u JPEGLosslessPredictors     = 517;
+    const int16u JPEGPointTransforms        = 518;
+    const int16u JPEGQTables                = 519;
+    const int16u JPEGDCTables               = 520;
+    const int16u JPEGACTables               = 521;
 }
 
 //---------------------------------------------------------------------------
 static const char* Tiff_Compression(int32u Compression)
 {
     switch (Compression)
     {
-        case     1 : return "Raw";
-        case     2 : return "CCITT Group 3";
-        case     3 : return "CCITT T.4";
-        case     5 : return "LZW";
-        case     6 : return "JPEG (TIFF v6)";
-        case     7 : return "JPEG (ISO)";
-        case     8 : return "Deflate";
-        case 32773 : return "PackBits";
-        default    : return ""; //Unknown
+    case 1: return "Raw";
+    case 2: return "CCITT T.4/Group 3 1D Fax";
+    case 3: return "CCITT T.4/Group 3 Fax";
+    case 4: return "CCITT T.6/Group 4 Fax";
+    case 5: return "LZW";
+    case 6: return "JPEG (TIFF v6.0)";
+    case 7: return "JPEG (ISO)";

```

Comment:
```suggestion
    case 6:
    case 7: return "JPEG (ISO)";
```

As we are here, I would like to remove the "style", the CodecID will say that and the idea is to fill the technical name.